File: src/main/java/de/maggiwuerze/xdccloader/events/bot/TargetBotCreationEvent.java
package de.maggiwuerze.xdccloader.events.bot;

import de.maggiwuerze.xdccloader.events.EntityCreationEvent;
import de.maggiwuerze.xdccloader.model.entity.Bot;

public class TargetBotCreationEvent extends EntityCreationEvent<Bot> {

	public TargetBotCreationEvent(Object source, Bot bot) {
		super(source, bot);
	}
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/events/bot/TargetBotCreationEvent.kt
package de.maggiwuerze.xdccwebloader.events.bot

import de.maggiwuerze.xdccwebloader.events.EntityCreationEvent
import de.maggiwuerze.xdccwebloader.model.entity.Bot

class TargetBotCreationEvent(source: Any, bot: Bot) : EntityCreationEvent<Bot>(source, bot)